tests/rt_utils.sh

```diff
@@ -270,9 +270,25 @@ check_results() {
         d=$( cmp ${RTPWD}/${CNTL_DIR}/$i ${RUNDIR}/$i | wc -l )
 
         if [[ $d -ne 0 ]] ; then
-          echo ".......NOT OK" >> ${REGRESSIONTEST_LOG}
-          echo ".......NOT OK"
-          test_status='FAIL'
+
+          if [[ ${MACHINE_ID} =~ orion || ${MACHINE_ID} =~ hera ]]; then
+            printf ".......NOT OK...TRY ALT CHECK.." >> ${REGRESSIONTEST_LOG}
+            printf ".......NOT OK...TRY ALT CHECK"
+            d=$( ${PATHRT}/compare_ncfile.py ${RTPWD}/${CNTL_DIR}/$i ${RUNDIR}/$i 2>/dev/null | wc -l )
```

Comment:
Just noticed this.
Why is a `wc -l` being done here.  One can operate on the return code of this script.
This can be replaced by:
```
python ${PATHRT}/compare_ncfile.py ${RTPWD}/${CNTL_DIR}/$i ${RUNDIR}/$i 
d=$?
```

Reply:
That won't work. Because of set -e, the script will exit. Can try piping the two lines.